src/sass/_shame.scss

```diff
@@ -138,3 +138,54 @@ body .row {
 .csp-inline-patch-logout {
   padding: 2em 0;
 }
+
+// /education/apply/ uses this expander button for navigation
+// Where else can I put this? Also, it could use a better name.
```

Comment:
edu-benefits.scss isn't used on `/education/apply`; that was the first thing I tried. The entry point for this page is `no-react.css`. I'm not sure how to load another css file from this markdown...